scripts/globals/pets/avatar.lua

```diff
@@ -0,0 +1,385 @@
+-----------------------------------
+-- PET: Avatar
```

Comment:
If the light spirit isn't an avatar, you could move the elemental spirit logic out into their own file

Reply:
So there's quite a few places i've seen that pet type is defined/used. I think in general summoner pets are all flagged as "avatar", so I piggybacked off that structure.